src/Queue.php

```diff
@@ -20,6 +20,9 @@
 
 final class Queue implements QueueInterface
 {
+    /** @psalm-suppress MissingClassConstType */
+    public const DEFAULT_CHANNEL_NAME = 'yii-queue';
```

Comment:
Let's move this constant to the interface, because it's a more common thing than a concrete implementation.

Reply:
done